content/blog/2022-07-FGBS1/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Fighting Galactic Bullshit 1"
+tease: "Dealing with misconceptions one blog post at a time"
+authors: "The Galaxy Team"
+date: "2022-07-11"
+---
+
+![galaxy hops, super smooth](hops.jpg)
+
+Galaxy is a free open platform designed to give researchers across the world unconstrained access to tools and infrastructure necessary for analysis of their data. Galaxy does a lot of things and contains many components, making it difficult to quickly comprehend everything that it can do. It also evolves [rapidly](https://docs.galaxyproject.org/en/master/releases/index.html). As a result, there are a number of misconceptions about what it is and what can (or cannot) be done with it. To spread the ultimate knowledge across the Universe, we are starting the **“Fighting Galactic Bullshit (FGBs)”** series (yes, we were definitely inspired by a certain [effort](https://www.callingbullshit.org/)). This series identifies the most common misconceptions and explains why they are, simply put, BULLSHIT! It will include the following “episodes”:
+
+1. Galaxy is only useful for genome scientists…, **But, all scientists have genomes, right?**
+2. Galaxy does not scale to large and complex problems…, **Define large and complex!**
+3. Galaxy offers nothing to informaticians who can write their own code…, **Are you sure?**
+4. Galaxy is hard to learn and hard to use…, **And bicycles are hard to park!**
+5. Galaxy is only useful for teaching, not real science…, **Who are bigger snobs: teachers or scientists?**
+6. Galaxy is neither popular with, nor widely used by real scientists…, **It depends on the meaning of the word “is”.**
```

Comment:
```suggestion
6. Galaxy is neither popular with, nor widely used by real scientists…, **It depends on the meaning of the words “popular” and "real".**
```
Again, not sure what will be in the blog post, but I feel like the meaning of `is` is not up for debate ?

Reply:
I like this 'clarification'

Reply:
you guys do not remember cultural references to bill Clinton trial, do you? 😄

Reply:
ah! now it rings a bell ;)